content/work/timedesigner.md

```diff
@@ -129,16 +126,16 @@ summary: |
 {% import 'embed.macros.njk' as embed %}
 {% import 'layout.macros.njk' as layout %}
 
-{{ quotes.find(collections.all, ['innovative', 'scheduling']) }}
+{{ quotes.find(collections.all, ['range', 'scheduling']) }}
 
 {{ layout.title('What We Did') }}
 
 {% call embed.media_block(
   media=embed.screen(
     notch='0',
     content=embed.img(
-      src='pages/timedesigner-phone.jpg',
-      alt='time designer screen',
+      src='work/timedesigner/mobile-lg.jpg',
```

Comment:
some images for these pages are in src/images/**work**/projectname/ while others are in src/images/**projects**/projectname

I'm curious, does anyone know why we have a reason or a set of criteria to know which work images go in which folder? Are projects more like the open source stuff? But if that is true, I think the preview image for the list pages only use the images from the project folder?

Reply:
Yeah, I'm confused too. And there are some images that aren't used at all (e.g. `timedesigner-mobile.jpg` -- which also isn't really "mobile").

Reply:
@jgerigmeyer yeah, that one is the one I replaced. I should have deleted it. I will do that now.